src/patternfly/base/tokens/tokens-local.scss

```diff
@@ -12,108 +12,64 @@
-    var(--pf-t--global--box-shadow--Y--sm--default) 
-    var(--pf-t--global--box-shadow--blur--sm) 
-    var(--pf-t--global--box-shadow--spread--sm--default)
-    var(--pf-t--global--box-shadow--color--sm--default);
-  --pf-t--global--box-shadow--sm--top: 
-    var(--pf-t--global--box-shadow--X--sm--top) 
-    var(--pf-t--global--box-shadow--Y--sm--top) 
-    var(--pf-t--global--box-shadow--blur--sm) 
-    var(--pf-t--global--box-shadow--spread--sm--directional)
-    var(--pf-t--global--box-shadow--color--sm--directional);
-  --pf-t--global--box-shadow--sm--bottom: 
-    var(--pf-t--global--box-shadow--X--sm--bottom) 
-    var(--pf-t--global--box-shadow--Y--sm--bottom) 
-    var(--pf-t--global--box-shadow--blur--sm) 
-    var(--pf-t--global--box-shadow--spread--sm--directional)
-    var(--pf-t--global--box-shadow--color--sm--directional);
-  --pf-t--global--box-shadow--sm--left: 
-    var(--pf-t--global--box-shadow--X--sm--left) 
-    var(--pf-t--global--box-shadow--Y--sm--left) 
-    var(--pf-t--global--box-shadow--blur--sm) 
-    var(--pf-t--global--box-shadow--spread--sm--directional)
-    var(--pf-t--global--box-shadow--color--sm--directional);
-  --pf-t--global--box-shadow--sm--right: 
-    var(--pf-t--global--box-shadow--X--sm--right) 
-    var(--pf-t--global--box-shadow--Y--sm--right) 
-    var(--pf-t--global--box-shadow--blur--sm) 
-    var(--pf-t--global--box-shadow--spread--sm--directional)
-    var(--pf-t--global--box-shadow--color--sm--directional);
+  --pf-t--global--box-shadow--sm: var(--pf-t--global--box-shadow--X--sm--default)
+    var(--pf-t--global--box-shadow--Y--sm--default) var(--pf-t--global--box-shadow--blur--sm)
+    var(--pf-t--global--box-shadow--spread--sm--default) var(--pf-t--global--box-shadow--color--sm--default);
+  --pf-t--global--box-shadow--sm--top: var(--pf-t--global--box-shadow--X--sm--top)
+    var(--pf-t--global--box-shadow--Y--sm--top) var(--pf-t--global--box-shadow--blur--sm)
+    var(--pf-t--global--box-shadow--spread--sm--directional) var(--pf-t--global--box-shadow--color--sm--directional);
+  --pf-t--global--box-shadow--sm--bottom: var(--pf-t--global--box-shadow--X--sm--bottom)
+    var(--pf-t--global--box-shadow--Y--sm--bottom) var(--pf-t--global--box-shadow--blur--sm)
+    var(--pf-t--global--box-shadow--spread--sm--directional) var(--pf-t--global--box-shadow--color--sm--directional);
+  --pf-t--global--box-shadow--sm--left: var(--pf-t--global--box-shadow--X--sm--left)
+    var(--pf-t--global--box-shadow--Y--sm--left) var(--pf-t--global--box-shadow--blur--sm)
+    var(--pf-t--global--box-shadow--spread--sm--directional) var(--pf-t--global--box-shadow--color--sm--directional);
+  --pf-t--global--box-shadow--sm--right: var(--pf-t--global--box-shadow--X--sm--right)
+    var(--pf-t--global--box-shadow--Y--sm--right) var(--pf-t--global--box-shadow--blur--sm)
+    var(--pf-t--global--box-shadow--spread--sm--directional) var(--pf-t--global--box-shadow--color--sm--directional);
```

Comment:
I'm not a huge fan or this put onto one line then broken up because of line length